src/components/Note/AddNote.jsx

```diff
@@ -0,0 +1,33 @@
+export const AddNote = () => {
+
+    return (
+        <>
```

Comment:
Is there a particular reason why you have used fragment here?
It does not make sense for me to use it here since this component already has a parent component <App>

Reply:
No reason I removed that as well